dbcon/mysql/ha_calpont_execplan.cpp

```diff
@@ -7872,6 +7867,8 @@ int getSelectPlan(gp_walk_info& gwi, SELECT_LEX& select_lex, SCSEP& csep, bool i
                 {
                     Item_int* select = (Item_int*)select_lex.master_unit()->global_parameters()->select_limit;
                     csep->limitNum(select->val_int());
+                    // MCOL-894 Activate parallel ORDER BY
+                    csep->orderByThreads(gwi.thd->variables.infinidb_orderby_threads);
```

Comment:
This may be something we can reasonably hide from the user to not create more vars or configurables.  Depending on the characteristics of the alg, a good default would be 1-2 * num_cores.

Reply:
When a user sets it to one CS uses original single-thread mode. This mode will be more efficient when sorting huge results sets w/o reasonable LIMIT(<1000000). One has to set the session variable to 1 to use original single-thread sorting.
We can eliminate this knob using three approaces:
1) Make the 2nd sorting phase parallel. Then parallel sorting will be faster under real life workloads.
2) When CS will have a statistics that allow us to predict result set to deduce that and automaticaly pick the proper sorting mode
3) If we have RowgroupDLs that can add subscribers on the fly we can add threads as need them thus not using parallel sorting for small datasets.